modules/pool/pool.service.ts

```diff
@@ -84,6 +85,10 @@ export class PoolService {
         return this.poolGqlLoaderService.getPools(args);
     }
 
+    public async getAggregatorPools(args: QueryPoolGetPoolsArgs): Promise<GqlPoolAggregator[]> {
+        return this.poolGqlLoaderService.getAggregatorPools(args);
+    }
+
```

Comment:
i'm thinking if it's not a good opportunity to refactor this service as well, make it similar to events, where we don't have a big service, but purpose specific functions that can be run independently.

Reply:
I thought we ended up agreeing that we'll reuse what we have and do a bigger refactor early next year when we investigate auto generated queries